### DISCUSSION TITLE
Redshift Data Sharing Not Visible in Database Navigator

### DISCUSSION THREAD

**Discussion by niraj8241** [dbeaver/dbeaver#13247] [Q&A] [answered]
Redshift has enabled data-sharing as a feature to share data across clusters. Currently, dbeaver does not show data sharing under the database navigator as all the other schemas/tables do. 
https://docs.aws.amazon.com/redshift/latest/dg/datashare-overview.html

There are tools like Aginity Pro that provide this feature. Here is how it looks like https://www.aginity.com/blog/how-to-use-amazon-redshift-data-sharing-in-aginity-pro-and-aginity-premium/

Can we have this enabled for DBeaver?

I am happy to provide more info. if required.

**Comment by uslss** [comment, marked as answer]
Seems it is already implemented in EE version. You could check it with [a trial version.](https://dbeaver.com/download/enterprise/)

**Comment by niraj8241** [comment]
@uslss I can confirm that i see the sharing in EE. Will this ever be a community edition feature?

**Comment by uslss** [comment]
@niraj8241 it is not planned at the moment
